src/aio/disk_engine.h

```diff
@@ -28,6 +28,7 @@
 
 #include "aio_provider.h"
 
+#include <dsn/tool_api.h>
```

Comment:
Why did you add `#include <dsn/tool_api.h>`?

Reply:
https://github.com/XiaoMi/rdsn/blob/5cf7373e27c0730a8ae7cc11b9780d23c3dc774d/src/aio/disk_engine.cpp#L40-L42 need it, otherwise compile error

Reply:
so you can add it in `disk_engine.cpp`, but not `disk_engine.h`

Reply:
Yes